tests/Frameworks/Laravel/Version_10_x/config/database.php

```diff
@@ -46,11 +46,11 @@
         'mysql' => [
             'driver' => 'mysql',
             'url' => env('DATABASE_URL'),
-            'host' => env('DB_HOST', '127.0.0.1'),
+            'host' => env('DB_HOST', 'mysql_integration'),
             'port' => env('DB_PORT', '3306'),
-            'database' => env('DB_DATABASE', 'forge'),
-            'username' => env('DB_USERNAME', 'forge'),
-            'password' => env('DB_PASSWORD', ''),
+            'database' => env('DB_DATABASE', 'test'),
+            'username' => env('DB_USERNAME', 'test'),
+            'password' => env('DB_PASSWORD', 'test'),
```

Comment:
I believe these should be set in the `.env`, but I ultimately don't care :)

Reply:
I don't disagree with having this on the `.env`. However I did it this way to keep consistency with the previous versions. Happy to change it if needed

Reply:
No that's alright :) It's working. As long as we have some "history of why" in this PR, that's ok for me